src/didactUriCompletionItemProvider.ts

```diff
@@ -302,4 +304,20 @@ export class DidactUriCompletionItemProvider implements vscode.CompletionItemPro
 		}
 		return null;
 	}
+
+	insertDidactBadgeAdoc() : vscode.CompletionItem {
+		const labelText = "Insert Didact Badge";
+		const snippetText = `image:https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/icon/poweredByDidact.png[Powered by Didact,link="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-didact"]`;
```

Comment:
is it possible to point to an image embedded in the extension?
it will allow having the image in airgapped environment.
it will avoid to break added badges if the structure of the git repository is modified.

If not possible, I think a test checkign that the image is still available at this place should be provided

Reply:
Yes, that's a possibility (using the extension-relative image)

I'm not sure about the light/dark background thing however, This is less about the theme and more about having a simple badge available to the user who wants it to be there. But definitely something we can chat with the community about.